src/raftstore/store/store.rs

```diff
@@ -1520,10 +1520,20 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             if peer.size_diff_hint < self.cfg.region_check_size_diff {
                 continue;
             }
-            info!("{} region's size diff {} >= {}, need to check whether should split",
-                  peer.tag,
-                  peer.size_diff_hint,
-                  self.cfg.region_check_size_diff);
+
+            match peer.approximate_size() {
```

Comment:
do we need to check this in the split check worker so we don't need to block the raft store thread if calculating size may be slow?